site/content/installation/ngf-images/jwt-token-docker-secret.md

```diff
@@ -40,6 +40,7 @@ You will need the following items from [MyF5](https://my.f5.com) for these instr
     kubectl get secret nginx-plus-registry-secret --output=yaml
     ```
 
+{{< include "installation/jwt-password-note.md" >}}
```

Comment:
Does this really need to be an include since it's only used in one place?

Reply:
Probably not, but it could be used elsewhere if JWT is mentioned in another document.

An include doesn't really affect the build time or size of documentation artefacts, and allows for content not specific to a particular page to be compartmentalised.

Reply:
We do include identical content in two other repositories. As we're working on changes, one of my (for now personal) goals is to push for some "universal" includes that works with all of our docs (and maybe even blogs).